### DISCUSSION TITLE
**我跟你爆了！！！**

### DISCUSSION THREAD

**Discussion by Socialm-an** [share121/inter-knot#426] [其他]
![2c50a07f3804401791054b4250e2a1fd_2Q==](https://github.com/user-attachments/assets/5225d6d1-dbc4-453f-9adb-4d571e26e389)
### **我跟你爆了！！！我跟你爆了！！！我跟你爆了！！！我跟你爆了！！！我跟你爆了！！！我跟你爆了！！！我跟你爆了！！！我跟你爆了！！！我跟你爆了！！！我跟你爆了！！！我跟你爆了！！！我跟你爆了！！！我跟你爆了！！！我跟你爆了！！！我跟你爆了！！！我跟你爆了！！！我跟你爆了！！！我跟你爆了！！！我跟你爆了！！！我跟你爆了！！！我跟你爆了！！！我跟你爆了！！！我跟你爆了！！！我跟你爆了！！！**

**Comment by Kamisato100** [comment]
火箭布，何时来的

**Comment by SaKura-071721** [comment]
嗯呢！嗯呢！

**Comment by kano0x0** [comment]
嗯呢呢呢！

**Comment by nahida0-o** [comment]
嗯呢！嗯呢哒！

**Comment by Friskdrink** [comment]
嗯呢，嗯呐达。

**Comment by maijiaosesu** [comment]
嗯呢哒，嗯呢哒（一起爆了！！！）

**Comment by yiwangzj** [comment]
让姨姨ruarua…等等，怎么我在天上？

**Comment by huyou111** [comment]
嗯呢，嗯呐达。

**Comment by Elysia-Ego-093** [comment]
好家伙，谁家的火箭布

**Comment by light814** [comment]
喵

**Comment by ProxyPhaeton** [comment]
我最懂爆炸了，这个委托让我来
